nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -657,18 +661,16 @@ class ConfigEncryptionTool {
     /**
      * Loads the flow definition from the provided file path, handling the GZIP file compression. Unlike {@link #loadLoginIdentityProviders()} this method does not decrypt the content (for performance and separation of concern reasons).
      *
-     * @return the file content
+     * @param The path to the XML file
+     * @return The file content
      * @throw IOException if the flow.xml.gz file cannot be read
      */
-    private String loadFlowXml() throws IOException {
+    private InputStream loadFlowXml(String flowXmlPath) throws IOException {
```

Comment:
The parameter `flowXmlPath` should probably be renamed since it shadows the static `flowXmlPath` variable declared in the class.